widgetssdk/src/main/java/com/glia/widgets/core/engagement/domain/ShouldShowEndEngagementDialogUseCase.java

```diff
@@ -0,0 +1,13 @@
+package com.glia.widgets.core.engagement.domain;
+
+public class ShouldShowEndEngagementDialogUseCase {
```

Comment:
A good usecase behaves like a pure function. It has no globally mutable state. Has only one public method, usually called `execute()`, `invoke()` etc and the arguments are passed through that method. This makes the usecase easy to understand, predictable and easily testable. 

Here in this situation, I think usecase is not needed at all.

Reply:
UseCase was removed.